src/settings/Settings.ts

```diff
@@ -586,4 +586,9 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Enable experimental, compact IRC style layout"),
         default: false,
     },
+    "dontShowChatEffects": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td("Don't show chat effects"),
```

Comment:
I think we prefer positively phrased settings,

e.g `Show chat effects` with default=true

Reply:
actually that was based on Nads suggestion, will change it

Reply:
In that case leave it, we are planning to go through all Settings "soon" and homogenize them anyhow